test/smoke/src/helpers/setupEnvironmentHelper.ts

```diff
@@ -45,6 +45,49 @@ export class SetupEnvironmentHelper {
         SetupEnvironmentHelper.patchMetroConfig(workspacePath);
     }
 
+    public static  prepareHermesReactNativeApplication(workspaceFilePath: string, resourcesPath: string, workspacePath: string, appName: string, customEntryPointFolder: string, version?: string) {
+        let commandClean = path.join(workspacePath, "android", "gradlew") + " clean";
```

Comment:
It is better to use spawn as it allows to add arguments to the executable command.